src/components/settings/settings-custom-labels/settings-custom-labels.test.js

```diff
@@ -153,11 +153,14 @@ describe('Given SettingsCustomLabels', () => {
         getByRole,
         getByTestId,
       } = renderSettingsCustomLabels();
+      const name = 'ui-eholdings.settings.customLabels.remove ' +
```

Comment:
Why do we need to join three translations together? What element will it match?

Reply:
`div` with `role='dialog'`, the debugger reveals this name.